addons/packages/vsphere-cpi/1.20.0/bundle/config/schema.yaml

```diff
@@ -0,0 +1,79 @@
+#! schema.yaml
+
+#@data/values-schema
+#@schema/desc "OpenAPIv3 Schema for vsphere-cpi"
+---
+vsphereCPI:
+  #@schema/desc "The cryptographic thumbprint of the vSphere endpoint's certificate"
+  tlsThumbprint: ""
+  #@schema/desc "The IP address or FQDN of the vSphere endpoint"
+  #@schema/nullable
```

Comment:
Server is required, should we remove this line
According to https://github.com/vmware-tanzu/community-edition/blob/9b6fe7b5c89fd4b87b575253c6c01eea5410c743/addons/packages/vsphere-cpi/1.20.0/README.md

Reply:
Yes, it is required. The assersion is done in `values.star`. Marking it nullable because the default value is null. If not marked, ytt will throw error.